packages/mobile/src/home/saga.ts

```diff
@@ -43,13 +45,6 @@ export function* refreshBalances() {
   yield put(fetchSentEscrowPayments())
 }
 
-export function* refreshBalancesWithLoadingSaga() {
```

Comment:
Removed because its unused and redundant with `watchRefreshBalances`

Reply:
good catch!